recipes/noiseflow/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ae0f060010667a924f7281976762d85cf5ba2402ccb404a0bab7a5fc893188fe
+
+build:
+  number: 0
+  skip: True  # [python_impl == 'pypy']
+  script: 
+    - build_osx.sh  # [osx]
+    - build_linux.sh  # [linux]
+    - build_win.bat  # [win]
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+
+requirements:
+  build:
+    - "{{ compiler('cxx') }}"
+    - python                              
+    - poetry-core
+    - setuptools                          
+    - cmake
+    - numpy
+    - pip  
+    - make  # [unix]
+    - ninja  
+
+  host:
+    - pip
+    - numpy >=1.25.0
+    - fftw  # [unix]
+    - pybind11  # [unix]
+    - xtensor  # [unix]
+    - xtensor-python  # [unix]
+    - xsimd =8.0.5  # [unix]
```

```suggestion
    - xsimd =8.0.5  # [unix]
```
It is not a good idea to enable simd: the hardware that you compile on and run on are not the same. So this will lead to errors

---

Have you performed the tests on different hardware, with different OS (version) ? If not, passing tests are expected

---

Not yet, I need to remove the `xsimd` I think.